docs/src/prototypes/VirtualizedTable/VirtualizedTable.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  gridCellBehavior,
+  gridHeaderCellBehavior,
+  gridNestedBehavior,
+  gridRowBehavior,
+  Table,
+} from '@fluentui/react'
+import * as React from 'react'
+import { AutoSizer, List as ReactVirtualizedList } from 'react-virtualized'
+import getItems from './itemsGenerator'
+
+const { rows } = getItems()
+
+const VirtualizedTablePrototype = () => {
+  const rowGetter = ({ index }) => {
+    return rows[index]
+  }
+
+  const accessibilityListProperties = {
```

Comment:
```suggestion
  const accessibilityListProperties: Partial<ListProps> = {
```

or 

```suggestion
  const accessibilityListProperties: ListProps = {
```

---

Also let's move out to upper scope
And let's add a description:
```
// Overrides ARIA attributes assigned by default that break accessibility
const accessibilityListProperties = {
```